lib/task.js

```diff
@@ -665,6 +668,24 @@ export default class Task {
       return await this.abortRun(this.taskState);
     }
     this.runtime.log('task run');
+
+    // Call started hook when container is started
+    dockerProc.once('container start', async (container) => {
+      try {
+        await this.states.started(this);
+      }
+      catch (e) {
+        debug(e.stack);
+        return await this.abortRun(
```

Comment:
Sure, we could, which would mean that we have mock/test features in features.js right?  Unless we offload that to somewhere else.  Also, that only tests the logic here of the 'started' state handling.

Although if the purpose of just testing the logic specifically there (the try/catch), then it would easily be tested just by doing what we do in other tests and specify an invalid key path which would test to make sure not only that the feature throws an exception when the key can't be found, but also that we handle it here properly.